src/main/java/com/mycodefu/vsssfshss/names/NameGenerator.java

```diff
@@ -0,0 +1,28 @@
+package com.mycodefu.vsssfshss.names;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.security.SecureRandom;
+import java.util.List;
+import java.util.Random;
+
+public class NameGenerator {
+    private static String[] adjectives = stringsFromTextResource(NameGenerator.class.getResourceAsStream("/names/adjectives.txt"));
+    private static String[] nouns = stringsFromTextResource(NameGenerator.class.getResourceAsStream("/names/nouns.txt"));
+    private static long seed = System.currentTimeMillis();
+
+    private static String[] stringsFromTextResource(InputStream resource) {
+        InputStreamReader inputStreamReader = new InputStreamReader(resource);
+        BufferedReader bufferedReader = new BufferedReader(inputStreamReader);
+        List<String> lines = bufferedReader.lines().toList();
+        return lines.toArray(new String[]{});
```

Comment:
This should close the streams before returning. Otherwise there is a resource leak.